cli/tauri.js/templates/src-tauri/Cargo.toml

```diff
@@ -19,6 +20,9 @@ phf = "0.7.24"
 includedir = "0.5.0"
 tauri = <%= tauriDep || `{ version = "0.2.0" }` %>
 
```

Comment:
Just noticed that phf and serde are still out of date.

Reply:
will update them.